src/components/ChatLog.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.js'
+
+const ChatLog = (props) => {
+    if (!props.chatData) {
+        return null
+    }
```

Comment:
We don't need this check here. If a value is undefined, React will just render a blank screen